packages/lib-auth/src/models/oauth/OAuthStore.spec.js

```diff
@@ -0,0 +1,39 @@
+import OAuthStore from './OAuthStore'
+
+const CREDENTIALS = {
+  expiresAt: 123456789,
+  token: 'foobar'
+}
+
+const DEFAULT_CREDENTIALS = {
+  expiresAt: 0,
+  token: null
+}
+
+let oauthStore
+
+describe.only('Model > OAuthStore', function () {
+  beforeEach(function () {
+    oauthStore = OAuthStore.create()
+  })
+
+  it('should exist', function () {
+    expect(OAuthStore).to.not.equal(undefined)
+  })
+
+  it('should have an `credentials` property', function () {
+    expect(oauthStore.credentials).to.not.equal(undefined)
```

Comment:
These expectations can be `expect(undefined).to.be.undefined;` However, I think the chai `exist` assertion might be better: http://www.chaijs.com/api/bdd/#method_exist, `expect(object).to.exist`, or best, actually checking what it should equal.

Reply:
Fair enough. I just wanted to check that they were defined, but checking their actual type is a much better idea.